CompanyEmployees.Core.Domain/Entities/Attendance.cs

```diff
@@ -0,0 +1,29 @@
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace CompanyEmployees.Core.Domain.Entities
+{
+    public class Attendance
+    {
+        [Column("AttendanceId")]
+        public Guid Id { get; set; }
+
+        [Required]
+        [ForeignKey(nameof(Employee))]
+        public Guid EmployeeId { get; set; }
+
+        [Required]
+        public DateTime CheckInTime { get; set; }
+
+        public DateTime? CheckOutTime { get; set; }
+
+        [Required]
```

Comment:
The `Status` property is marked as `[Required]` but is also declared nullable (`string?`). This can cause confusion and may defeat the purpose of the validation annotation.

**Suggested change**
```suggestion
public string Status { get; set; }
```